src/Adaptor.js

```diff
@@ -130,7 +131,7 @@ function handleOptions(options) {
   if (options && options.setNull === false) {
     return false;
   }
-  return (options && options.setNull) || "'undefined'";
```

Comment:
Hey @lakhassane , I missed this during our conversation earlier. it _seems_ that this actually is meant to allow the user to define a value in `options.setNull` which tells our parser to replace that specific value with `NULL`. I'm going to play around with this a bit before merging—I'd like to preserve that contract.

Reply:
@taylodowns2000  In this case we can keep `return (options && options.setNull) || "'undefined'";`

I tried playing with it to use `setNull:1001` on the state and it worked fine.